Cushing.md

```diff
@@ -0,0 +1,5 @@
+#Cushing, MN
+
```

Comment:
Wasn't there a few episodes of Supernatural that were supposed to be in Cushing?